src/routes/webhooks/paddle.ts

```diff
@@ -0,0 +1,105 @@
+import { FastifyInstance } from 'fastify';
+import {
+  Environment,
+  EventName,
+  Paddle,
+  SubscriptionCanceledEvent,
+  SubscriptionCreatedEvent,
+} from '@paddle/paddle-node-sdk';
+import createOrGetConnection from '../../db';
+import { updateSubscriptionFlags } from '../../common';
+import { User } from '../../entity';
+import { logger } from '../../logger';
+
+const paddleInstance = new Paddle(process.env.PADDLE_API_KEY, {
+  environment: Environment.sandbox,
+});
+
+const planTypes = {
+  pri_01jcdp5ef4yhv00p43hr2knrdg: 'monthly',
+  pri_01jcdn6enr5ap3ekkddc6fv6tq: 'yearly',
+};
+
+const updateUserSubscription = async ({
+  data,
+  state,
+}: {
+  data: SubscriptionCreatedEvent | SubscriptionCanceledEvent | undefined;
+  state: boolean;
+}) => {
+  if (!data) {
+    return;
+  }
+  const con = await createOrGetConnection();
+  const userId = data.data?.customData?.user_id;
+  if (!userId) {
+    logger.error('User ID missing in payload');
```

Comment:
can we prefix everything with `paddle` or add `{ type: 'paddle' }` so we can easily search through logs?

Reply:
Yeah, think it's a good idea.
I added prefixes to the top reader cron so it's easier to locate logs

Reply:
(kinda miss the logging in python)

Reply:
Good id, will add prefix